src/stores/filings.ts

```diff
@@ -159,7 +159,7 @@ export const useBcrosFilings = defineStore('bcros/filings', () => {
       isFutureEffective: header.isFutureEffective,
       name: header.name,
       status: header.status,
-      submittedDate: apiToUtcString(header.date),
+      submittedDate: header.paymentDate || apiToUtcString(header.date),
```

Comment:
Payment date is already "FormattedDateTimeGmt", which is what apiToUtcString returns for header date.